src/org/gk/qualityCheck/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities.java

```diff
@@ -0,0 +1,102 @@
+package org.gk.qualityCheck;
+
+import org.gk.database.InstanceListPane;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+
+import java.util.*;
+
+/**
+ * Flags human ReactionlikeEvents that do not have a populated 'disease' attribute, but have non-human participants.
+ */
+
+public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities extends AbstractQualityCheck {
+
+
+    @Override
+    public QAReport checkInCommand() throws Exception {
+        QAReport report = new QAReport();
+        MySQLAdaptor dba = (MySQLAdaptor) dataSource;
+        QACheckUtilities.setSkipList(Arrays.asList("168249")); // Innate Immunity Pathway
+        QACheckUtilities.setHumanSpeciesInst(dba);
+
+        Collection<GKInstance> reactions = dba.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent);
+        for (GKInstance reaction : reactions) {
+            // isHumanDatabaseObject checks that the species attribute only contains a Homo sapiens species instance. Multi-species RlEs are excluded.
+            if (!QACheckUtilities.memberSkipListPathway(reaction)
+                    && QACheckUtilities.isHumanDatabaseObject(reaction)
+                    && reaction.getAttributeValue(ReactomeJavaConstants.disease) == null) {
```

Comment:
I notice this check has been used a couple of times.  Maybe it could get a named method in QACheckUtilities:

```
public boolean hasDiseaseTag(GKInstance instance) {
    return instance.getAttributeValue(ReactomeJavaConstants.disease) != null;
}
```
and then just add a "!" to check for the absence of disease.  

For me, having a name for the what the condition is checking makes it clearer at a glance.

Reply:
Yea I did consider that as well at the time, I'll implement that